### PR TITLE
add glow package to wolfi

### LINKED ISSUE #13254: [Wolfi Package Request]: mods and glow
### What software would you like us to add to wolfi-os. Ideally include a URL to the project and its source.

https://github.com/charmbracelet/mods
https://github.com/charmbracelet/glow

mods lets you pipe commands through openai/localai via the command line or a pipeline. glow then formats the results into markdown. 

### which versions of the software should we include?

latest

### Add some justification for why this specific package and versions are important.

This is admittedly demo candy. 😄 

### Please check all that apply

- [X] This package has an un-restrictive license
- [X] The package/versions proposed are actively maintained upstream
- [ ] I am interested in adding this package to Wolfi OS myself
- [ ] I am willing to help maintain this package

### PR DESCRIPTION
**Pull request by kranurag7**
Fixes: https://github.com/wolfi-dev/os/issues/13254

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: glow.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - uses: git-checkout
+    with:
+      repository: https://github.com/charmbracelet/glow
+      tag: v${{package.version}}
+      expected-commit: ad21129703d0bd48a9b45227f56d2737623db57b
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.17.0 golang.org/x/crypto@v0.17.0
+      replaces: github.com/dgrijalva/jwt-go=github.com/golang-jwt/jwt/v5@v5.2.0 github.com/golang-jwt/jwt=github.com/golang-jwt/jwt/v5@v5.2.0
```
Comment (kranurag7) [inline glow.yaml:28]: I was not able to fix it up even after the replace block. 
```bash
[sdk] ❯ wolfictl scan ./packages/x86_64/glow-1.5.1-r0.apk 
🔎 Scanning "./packages/x86_64/glow-1.5.1-r0.apk"
└── 📄 /usr/bin/glow
        📦 github.com/dgrijalva/jwt-go v3.2.0+incompatible (go-module)
            High CVE-2020-26160 GHSA-w73w-5m7g-f7qc
```

Comment (kranurag7) [reply glow.yaml]: fixed via https://github.com/kranurag7/os/commit/02fab311454c53d3ea7207b42973c867210cd5e7

**Comment by kranurag7** [comment]
I'm marking this PR as ready to review now given things are looking good locally. 
```bash
[sdk] ❯ wolfictl scan ./packages/aarch64/glow-1.5.1-r0.apk
🔎 Scanning "./packages/aarch64/glow-1.5.1-r0.apk"
✅ No vulnerabilities found
[sdk] ❯ apk add --allow-untrusted ./packages/aarch64/glow-1.5.1-r0.apk
(1/1) Installing glow (1.5.1-r0)
OK: 1092 MiB in 67 packages
[sdk] ❯ glow --version
glow version 1.5.1 (ad21129)
[sdk] ❯ make test/glow
# truncated output
2024/08/19 19:13:29 WARN + '[' -d /home/build ]
2024/08/19 19:13:29 WARN + cd /home/build
2024/08/19 19:13:29 WARN + glow --version
2024/08/19 19:13:29 INFO glow version 1.5.1 (ad21129)
2024/08/19 19:13:29 WARN + exit
```

File: glow.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - gobump
+  environment:
+    CGO_ENABLED: "0"
```
Comment (mamccorm) [inline glow.yaml:17]: This can be removed as go/build handles all these dependencies now:

```suggestion
```

Comment (kranurag7) [reply glow.yaml:17]: Thanks Mark, I should have reviewed the pipeline first. (will do it next time onward)
https://github.com/chainguard-dev/melange/blob/41f30e3eaad19341d38ad2e7a13faf2e501f7d2d/pkg/build/pipelines/go/build.yaml#L3-L7

**Review by mamccorm — APPROVED** [review]

**Pull request merged**